[2022-02-15 03:14:41,237] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:14:41,259] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:14:41,260] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:14:41,261] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:14:41,262] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:14:41,283] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.stg_customers> on 2022-02-15 03:14:06.780838+00:00
[2022-02-15 03:14:41,290] {standard_task_runner.py:52} INFO - Started process 1194 to run task
[2022-02-15 03:14:41,302] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.stg_customers', 'manual__2022-02-15T03:14:06.780838+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpao95hndm', '--error-file', '/tmp/tmp51eellro']
[2022-02-15 03:14:41,305] {standard_task_runner.py:77} INFO - Job 108: Subtask model.my_new_project.stg_customers
[2022-02-15 03:14:41,461] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.stg_customers manual__2022-02-15T03:14:06.780838+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:14:41,605] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.stg_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:14:06.780838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:14:06.780838+00:00
[2022-02-15 03:14:41,611] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:14:41,613] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models stg_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:14:41,659] {subprocess.py:85} INFO - Output:
[2022-02-15 03:14:49,397] {subprocess.py:89} INFO - 03:14:49  Running with dbt=1.0.1
[2022-02-15 03:14:49,603] {subprocess.py:89} INFO - 03:14:49  Unable to do partial parsing because config vars, config profile, or config target have changed
[2022-02-15 03:14:53,689] {subprocess.py:89} INFO - 03:14:53  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:14:53,693] {subprocess.py:89} INFO - 03:14:53
[2022-02-15 03:14:57,089] {subprocess.py:89} INFO - 03:14:57  Concurrency: 4 threads (target='dev')
[2022-02-15 03:14:57,093] {subprocess.py:89} INFO - 03:14:57
[2022-02-15 03:14:57,132] {subprocess.py:89} INFO - 03:14:57  1 of 1 START view model dbt_bytecode_training.stg_customers..................... [RUN]
[2022-02-15 03:14:59,284] {subprocess.py:89} INFO - 03:14:59  1 of 1 OK created view model dbt_bytecode_training.stg_customers................ [SUCCESS 1 in 2.09s]
[2022-02-15 03:14:59,326] {subprocess.py:89} INFO - 03:14:59
[2022-02-15 03:14:59,328] {subprocess.py:89} INFO - 03:14:59  Finished running 1 view model in 5.61s.
[2022-02-15 03:14:59,343] {subprocess.py:89} INFO - 03:14:59
[2022-02-15 03:14:59,350] {subprocess.py:89} INFO - 03:14:59  Completed successfully
[2022-02-15 03:14:59,356] {subprocess.py:89} INFO - 03:14:59
[2022-02-15 03:14:59,361] {subprocess.py:89} INFO - 03:14:59  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:15:00,408] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:15:00,539] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.stg_customers, execution_date=20220215T031406, start_date=20220215T031441, end_date=20220215T031500
[2022-02-15 03:15:00,649] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:15:00,761] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
